docs/ref-manual/portal/secrets.md

```diff
@@ -0,0 +1,134 @@
+Special Portal URLs
+===================
+
+When you run the Certora CLI, the tool outputs a link to the generated
+verification report.  By modifying this link, you can access additional files
+that the tool generates.  These are primarily intended for the tool developers,
+but they can be useful for users too in some cases.  This page lists these
+outputs and explains how to read them.
+
+
+Job Status
+----------
+
+The job status page contains information about the job, including the arguments
+that were passed to the command line, the job message, and the exact certora
+prover version.  It is created as soon as the job is submitted, and also shows
+the current status (whether the job has started or finished processing).
+
+To access the job status page, change the `output`  component of the URL to
+`jobStatus`.  For example,
+
+```
+https://vaas-stg.certora.com/output/65540/270dce9623d492937d82/?anonymousKey=6edb93d7abae7871f1c3be3b10863d64d2d72fef
+                             ──────
+```
+
+becomes
+
+```
+https://vaas-stg.certora.com/jobStatus/65540/270dce9623d492937d82/?anonymousKey=6edb93d7abae7871f1c3be3b10863d64d2d72fef
+                             ─────────
+```
+
+FinalResults.html and Results.txt
+---------------------------------
+
+The `FinalResults.html` is an older version of the verification report, but
```

Comment:
Maybe put a nice note box saying Certora is working on newer report formats that will make these ones obsolete

Reply:
Fixed